static/css/com/nav/global-nav.scss

```diff
@@ -9,35 +9,57 @@
   position: relative;
   float: right;
   font-size: 0;
-  margin-top: 2px;
+  height: 100%;
```

Comment:
What is problem it resolve?

Reply:
Fixed it, now use  `align-items: stretch`.